lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2352,6 +2352,19 @@ static void createConstTensorAttrsOnAccel(
   operands.push_back(ourCst);
 }
 
+static void createConstTensorAttrsOnAccel(
+    unsigned constVal, SILType valTy, SILLocation loc, ASTContext &ctx,
+    SILBuilder &B, std::string &opName,
```

Comment:
some of these arguments are unused.

Reply:
Ack. this is addressed in the next PR https://github.com/apple/swift/pull/17827/files